frontend/src/app/spot/components/tooltip/stories/Tooltip.stories.mdx

```diff
@@ -65,3 +67,23 @@ export const Template = (args) => ({
     {Template.bind({})}
   </Story>
 </Canvas>
+
+
+## Behaviour
+
+The tooltip appears next to the item being hovered (which must never be covered), but does not follow the cursor. Hovering out of the element that triggered it will cause the tooltip disappear.
+
+Tooltips automatically align themselves to the top or the bottom of the element it is attached to, depending on the positioning of the element in the viewport.
+
+## Margins and Spacing
+
+A tooltip's width is generally determined by the containing text.
+
+However, due to technical limitations, there is a hard min-width of 20-rem and a max-width of 90vw. Tooltips shorter than 20-rem will have additional space on the rigth.
+
+The minimum width of the tooltip depends on the length of the text, and the maximum width depends on the width of the containing element.
+
+A tooltip can be multi-line if necessary (?) It is never truncated.
```

Comment:
I think this is a correct assumption. A tooltip might even content a paragraph if needed. Having a truncated extra information text doesn't make to much sense in my opinion.